game/protobuf_include.vpc

```diff
@@ -1,51 +1,51 @@
-//-----------------------------------------------------------------------------
-//	protobuf_include.vpc
-//
-//	Project Script
-//-----------------------------------------------------------------------------
-
-$MacroRequired "PLATFORM"
-
-$Project
-{
-	$Folder "Link Libraries" [!$VS2015]
-	{
-		$Libexternal $SRCDIR\thirdparty\protobuf-2.6.1\bin\osx32\libc++\libprotobuf [$OSX32]
-		$Libexternal $SRCDIR\thirdparty\protobuf-2.5.0\vsprojects\Release\libprotobuf [$WIN32]
-		$Libexternal $SRCDIR\thirdparty\protobuf-2.5.0\src\.libs\libprotobuf [$LINUX32]
-	}
-
-	$Folder "Link Libraries" [$WIN64&&!$VS2015]
-	{
-		$File "$SRCDIR\thirdparty\protobuf-2.5.0\vsprojects\Release\libprotobuf.lib"
-		{
-			$Configuration "Debug" 	{ $ExcludedFromBuild "Yes" }
-		}
-		$File "$SRCDIR\thirdparty\protobuf-2.5.0\vsprojects\Debug\libprotobuf.lib"
-		{
-			$Configuration "Release" { $ExcludedFromBuild "Yes" }
-		}
-	}
-
-	$Folder "Link Libraries" [$VS2015]
-	{
-		$File "$SRCDIR\thirdparty\protobuf-2.5.0\vsprojects\Release\libprotobuf.lib" [$WIN32]
-		{
-			$Configuration "Debug" 	{ $ExcludedFromBuild "Yes" }
-		}
-		$File "$SRCDIR\thirdparty\protobuf-2.5.0\vsprojects\Debug\libprotobuf.lib" [$WIN32]
-		{
-			$Configuration "Release" { $ExcludedFromBuild "Yes" }
-		}
-
-		$File "$SRCDIR\thirdparty\protobuf-2.5.0\vsprojects\Release\libprotobuf.lib" [$WIN64]
-		{
-			$Configuration "Debug" 	{ $ExcludedFromBuild "Yes" }
-		}
-		$File "$SRCDIR\thirdparty\protobuf-2.5.0\vsprojects\Debug\libprotobuf.lib" [$WIN64]
-		{
-			$Configuration "Release" { $ExcludedFromBuild "Yes" }
-		}
-	}
-}
-
+//-----------------------------------------------------------------------------
+//	protobuf_include.vpc
+//
+//	Project Script
+//-----------------------------------------------------------------------------
+
+$MacroRequired "PLATFORM"
+
+$Project
+{
+	$Folder "Link Libraries" [!$VS2015]
+	{
+		$Libexternal $SRCDIR\thirdparty\protobuf-2.5.0\bin\osx32\libc++\libprotobuf [$OSX32]

```

Comment:
While this in particular is fine, please keep OSX changes out of this PR

Reply:
Will do.